contracts/Datastore.sol

```diff
@@ -518,4 +521,69 @@ contract Datastore is AragonApp {
         permissions.removeGroupFromFile(_fileId, _groupId);
         emit GroupPermissionsRemoved(msg.sender);
     }
+
+    /**
+     * @notice Add a label to the datastore
+     * @param _name Name of the label
+     * @param _color Color of the label
+     */
+    function createLabel(bytes28 _name, bytes4 _color) external {
+        labelList.createLabel(_name, _color);
```

Comment:
We should make sure only the users with `DATASTORE_MANAGER_ROLE` can create and delete labels